tests/AmyrisBio.Tests/testPrimer.fs

```diff
@@ -67,4 +66,173 @@ type TestPrimer() = class
+              "AAAAAATCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "AAAAAATCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "AAAAAATCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "AAAAAATCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "AAAAAATCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "AAAAAATCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "AAAAATCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "AAAATCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "AAATCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "AATCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "ATCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "TCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "CTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "TTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "TAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "AATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "ATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "TAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "AGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATATATATATATATACATATATAAAG" ;
+              "GATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATACATATATAAAG" ;
+              "ATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATACATATATAAAG" ;
+              "TTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATACATATATAAAG" ;
+              "TAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATACATATATAAAG" ;
+              "AATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATACATATATAAAG" ;
+              "ATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATACATATATAAAG" ;
+              "TTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATACATATATAAAG" ;
+              "TTAAACAGTATATGTACAGTTTTATATATATATATATATATATATACATATATAAAG" ;
+              "TAAACAGTATATGTACAGTTTTATATATATATATATATATATATACATATATAAAG" ;
+              "AAACAGTATATGTACAGTTTTATATATATATATATATATATATACATATATAAAG" ;
+              "AAAAAATCTTAATAGATTAATTTAAACAGTATATGTACAGTTTTATATATATATATATATATATATACATATATAAAG" ;
+            ]
+        let pen= {
```

Comment:
There is a lot of code duplication between these two test suites; it would be great to extract this record to the test class, as well as the `let task = ...` expression below.

Reply:
deduped code and the shared templates